components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -18,23 +18,22 @@ const MAX_INPUT_CHAR = 2000
 const CHAR_LIMIT_THRESHOLD = MAX_INPUT_CHAR * 0.80
 
 function InputBox () {
-  const [inputText, setInputText] = React.useState('')
   const context = React.useContext(DataContext)
 
-  const isCharLimitExceeded = inputText.length >= MAX_INPUT_CHAR
-  const isCharLimitApproaching = inputText.length >= CHAR_LIMIT_THRESHOLD
+  const isCharLimitExceeded = context.inputText.length >= MAX_INPUT_CHAR
+  const isCharLimitApproaching = context.inputText.length >= CHAR_LIMIT_THRESHOLD
 
   const onInputChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
-    setInputText(e.target.value)
+    context.setInputText(e.target.value)
   }
 
   const submitInputTextToAPI = () => {
-    if (!inputText) return
+    if (!context.inputText) return
     if (isCharLimitExceeded) return
     if (context.shouldDisableUserInput) return
 
-    getPageHandlerInstance().pageHandler.submitHumanConversationEntry(inputText)
-    setInputText('')
+    getPageHandlerInstance().pageHandler.submitHumanConversationEntry(context.inputText)
```

Comment:
we should probably also move these calls to the context, shouldn't we?

Reply:
moved to context